src/adapters/bidderFactory.js

```diff
@@ -288,12 +290,17 @@ export function newBidder(spec) {
           onResponse();
 
           function addBidUsingRequestMap(bid) {
-            const bidRequest = bidRequestMap[bid.requestId];
-            if (bidRequest) {
-              const prebidBid = Object.assign(bidfactory.createBid(STATUS.GOOD, bidRequest), bid);
-              addBidWithCode(bidRequest.placementCode, prebidBid);
+            // In Prebid 1.0 all the validation logic from bidmanager will move here, as of now we are only validating new params so that adapters dont miss adding them.
+            if (hasValidKeys(bid)) {
```

Comment:
Careful here...
Careful here... we've been telling adapters _not_ to add `bidderCode`, because it breaks aliasing. The `bidderFactory` should be doing that for them.

But... that happens a few lines after this: `Object.assign(bidfactory.createBid(STATUS.GOOD, bidRequest), bid);`.

So I don't think should be make sure that `bidderCode` exists here. If anything, we should make sure that it _doesn't_ exist yet.